myuw/resources/sws/file/student/v5/notice/FE36CCB8F66711D5BE060004AC494F31.json

```diff
@@ -144,8 +144,7 @@
 			"NoticeAttributes": [], 
 			"NoticeCategory": "StudentDAD", 
 			"NoticeContent": "<span class=\"notice-title\">Tuition Due</span><span class=\"notice-body-with-title\">Payment of tuition and tuition-related fees for winter quarter is due <span class=\"date\"></span>. For details, <a href=\"https://sdb.admin.washington.edu/students/UWNetID/tuition.asp\">view your Tuition Statement</a>.</span>", 
-			"NoticeType": "TuitDue",
-      "Critical": true
```

Comment:
I understand that this is mock data, but why did the criticality get removed here?

Reply:
We don't read this element. It is not supposed to be here.